api/tests/test_asset_api.py

```diff
@@ -386,4 +387,35 @@ def test_asset_allocation_history_has_assigner(self, mock_verify_id_token):
             HTTP_AUTHORIZATION="Token {}".format(self.token_user),
         )
         self.assertIn("assigner", response.data["allocation_history"][0].keys())
-        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    @patch("api.authentication.auth.verify_id_token")
+    def test_get_assets_by_logged_in_user(self, mock_verify_id_token):
+        mock_verify_id_token.return_value = {"email": self.user.email}
+        user = AssetOwner.objects.create(user=self.user)
+        Asset.objects.create(
+            asset_code="IC0014532",
+            serial_number="SN50123455",
+            purchase_date="2018-07-10",
+            model_number=self.assetmodel,
+            asset_location=self.centre,
+            department=self.department,
+            verified=True,
+            owned_by=user,
+        )
+
+        Asset.objects.create(
+            asset_code="IC001453",
+            serial_number="SN5012345",
+            purchase_date="2018-07-10",
+            model_number=self.assetmodel,
+            asset_location=self.centre,
+            department=self.department,
+            verified=True,
+        )
+        response = client.get(
+            "{}".format(reverse("assets-me")),
```

Comment:
I'll look to restructure the tests. I feel it would be neater and more readable to use f-strings for our string concatenations.